target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -794,6 +794,11 @@ tl-wa801nd-v3)
 	ucidef_set_led_netdev "lan" "LAN" "tp-link:green:lan" "eth1"
 	ucidef_set_led_wlan "wlan" "WLAN" "tp-link:green:wlan" "phy0tpt"
 	;;
+tl-wa801nd-v4)
+	ucidef_set_led_wlan "wlan" "WLAN" "tp-link:green:wlan" "phy0tpt"
+	ucidef_set_led_netdev "lan" "LAN" "tp-link:green:lan" "eth1"
+	ucidef_set_led_power "power" "POWER GREEN" "tp-link:green:power" "1"
```

Comment:
Could be merged with v3 and condition for power part.

Reply:
You'r right, i will merge this.